<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0">
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEditDniActualizar"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="textEditTelefonoActualizar"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEditApellidosActualizar"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEditNombreActualizar"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Apellidos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Dni</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEditEmailActualizar"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonActualizarCliente">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEliminarCliente">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
